types/react-maskedinput/index.d.ts

```diff
@@ -1,31 +1,26 @@
-// Type definitions for react-maskedinput 3.3
+// Type definitions for react-maskedinput 4.0
 // Project: https://github.com/insin/react-maskedinput
-// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions by: Karol Janyst <https://github.com/LKay>,
+//     Adam Lavin <https://github.com/lavoaster>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import { ComponentClass, HTMLAttributes } from "react";
+import * as React from "react";
 
-export as namespace MaskedInput;
-
-export = MaskedInput;
-
-declare const MaskedInput: MaskedInput;
-type MaskedInput = ComponentClass<MaskedInput.MaskedInputProps>;
-
-declare namespace MaskedInput {
-    interface FormatCharacter {
-        validate(char: string): string;
-        transform?(char: string): string;
-    }
+interface ReactMaskedInputFormatCharacter {
+    validate(char: string): string;
+    transform?(char: string): string;
+}
 
-    interface CharsFormatters {
-        [char: string]: FormatCharacter;
-    }
+interface ReactMaskedInputCharsFormatters {
```

Comment:
I assumed this would be fine with the potential breaking change introduced in 4.0.0. Will revert though.